easybuild/easyconfigs/b/Bazel/Bazel-7.3.2-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,29 @@
+name = 'Bazel'
+version = '7.3.2'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/bazelbuild/%(namelower)s/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+
+checksums = [
+    {'bazel-7.3.2-dist.zip': '8c24490a6445b00eb76a04adbb0172f5c51b1edbaeeef91ff7f3c7e86c7921ff'},
```

Comment:
not sure if it makes send, but I have it as 
```python
{'bazel-%(version)s-dist.zip': '8c24490a6445b00eb76a04adbb0172f5c51b1edbaeeef91ff7f3c7e86c7921ff'},
```

Reply:
Other than that, I reached the exact same easyconfig and it works for me on intel and gh200 aarch64

Reply:
will it make sense to use `%(namelower)s-%(version)s-dist.zip`?

Reply:
I would keep it as is then